addons/GearSwap/libs/ccConfig.lua

```diff
@@ -0,0 +1,30 @@
+-- Jobs you want to execute with, recomment put all active jobs you have lua for will look for <job>.lua or <playername>_<job>.lua files
+ccjobs = { 'BLM', 'BLU', 'BRD', 'BST', 'DRG', 'GEO', 'MNK', 'NIN', 'PLD', 'RDM', 'RNG', 'RUN', 'SAM', 'SCH', 'SMN', 'THF', 'WAR', 'WHM' }
+-- Put any items in your inventory here you don't want to show up in the final report
+-- recommended for furniture, food, meds, pop items or any gear you know you want to keep for some reason
+-- supports lua regex syntax, put each separate one as its own entry in the table
+-- see: https://www.lua.org/pil/20.2.html

```

Comment:
I would suggest using wildcard matching instead of Lua's string pattern matching. You would need to use the `windower.wc_match` function instead of `string.match` below, but people are more familiar with it and it's generally simpler to use (you actually used it by accident on `"Deed of*"` below, which is not a valid Lua string pattern (or well, it is, but probably not the one you intended it to be). The wildcard matching is in many cases also more powerful, since it allows alternation with `|`, which Lua string patterns do not.